frontend/jest/featureFlags.js

```diff
@@ -1,7 +1,36 @@
 const TEST_URL = require("../tests/common/constants").TEST_URL;
 
 module.exports = {
-  cookies: [],
+  cookies: [
```

Comment:
Awesome idea! I bet we can one up it by checking for a `CXG_COOKIE` param and automatically adding the element to the array

Reply:
ooh nice great idea!! 🔥 🔥 🔥 Will add that!

Reply:
Done! PTAL again thank you!